build/pipelines/templates/build-console-steps.yml

```diff
@@ -90,9 +132,22 @@ steps:
     flattenFolders: true
   condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
 
+- task: CopyFiles@2
+  displayName: 'Copy outputs needed for test runs to Artifacts'
+  inputs:
+    Contents: |
+     $(Build.SourcesDirectory)/bin/$(BuildPlatform)/$(BuildConfiguration)/*.exe
+     $(Build.SourcesDirectory)/bin/$(BuildPlatform)/$(BuildConfiguration)/*.dll
+     $(Build.SourcesDirectory)/bin/$(BuildPlatform)/$(BuildConfiguration)/*.xml
+     **/Microsoft.VCLibs.*.appx
+     **/TestHostApp/*
+    TargetFolder: '$(Build.ArtifactStagingDirectory)/$(BuildConfiguration)/$(BuildPlatform)/test'
+    OverWrite: true
+    flattenFolders: true
+  condition: and(and(succeeded(), eq(variables['BuildPlatform'], 'x64')), not(eq(variables['Build.Reason'], 'PullRequest')))
+
 - task: PublishBuildArtifacts@1
-  displayName: 'Publish Artifact (appx) (Non-PR builds only)'
+  displayName: 'Publish All Build Artifacts'
   inputs:
-    PathtoPublish: '$(Build.ArtifactStagingDirectory)/appx'
-    ArtifactName: 'appx-$(BuildConfiguration)'
-  condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
+    PathtoPublish: '$(Build.ArtifactStagingDirectory)'
+    ArtifactName: 'drop'
```

Comment:
gotta be careful with this one! x86 and x64 and arm builds will all stomp the hecc out of eachother this way, won't they? we're doing directory flattening...

Reply:
or, the build will fail b/c multiple arms are producing the same artifact... right? idk

Reply:
oh, so you moved them into a subfolder... so i don't hate that. the msix packages are also properly badged with their architecture

it's really okay with multiple arms publishing the same artifact?

Reply:
I believe it is really OK to publish the same artifact as much as you want. As far as I can gather, that's how MUX manages and they have a lot more config/publishing than us.